### ISSUE TITLE
UISP Migration to uisp.berlin.freifunk.net

### ISSUE THREAD

**Issue by Noki** [freifunk-berlin/meta#132]
With the rollout of babel routing and bgpdisco we changed the host suffix from `.olsr` to `.ff`. Due to the change devices can no longer reach the host and need to be migrated to another host. In order to make the uisp host a stable endpoint we should migrate it to `uisp.berlin.freifunk.net`.

- [x] Add dns entries to resolte `uisp.berlin.freifunk.net` to the correct IP addresses (@nicolasberens)
  - [x] Use IPv4 & IPv6 addresses reachable from the Internet so we can have a proper SSL certificate
- [x] Setup migration in UISP to the new host (@Noki)
- [x] Add a static hostname entry for `uisp.olsr` so devices can reconnect and the migration can take place (@Noki)

**Comment by nicolasberens** [comment]
77.87.50.24
2001:bf7:b301::24

dns might take a bit

please make sure only services that are necessary are accessible from the outside

**Comment by pktpls** [comment]
UISP is now part of our public attack surface - would be good to remove at least the gateway locations from it, if they're in there

**Comment by pktpls** [comment]
Or maybe we can set it up so it's only accessible from within the mesh

**Comment by Noki** [comment]
Restricting it sounds like the way to go. The idea behind the public IP and the subdomain is that those are more stable than our internal IPs / hostnames and could also be secured, but we do not really access from outside our network as everybody knows to use a jump host anyway. However it would be nice to find a way to have a valid Let's encrypt certificate that could also renew.

**Comment by Noki** [comment]
Everything is up and running now on the new host.

**Comment by pktpls** [comment]
> Or maybe we can set it up so it's only accessible from within the mesh

This is still to do - if UISP gets popped we can all take our yearly vacation and rebuild the whole network

**Comment by nicolasberens** [comment]
Let me take a look at it

**Comment by Noki** [comment]
Maybe we can use Caddy or something as a reverse proxy with authentication for external IPs?

**Comment by pktpls** [comment]
> I think you can still socksproxy to some freifunk node or gateway and still use it?

Yeah or run a VM and do SSH port forwarding for port 80/443 (I do that often)

> Restricting Access to the following networks would be a good start imho:

And `/.well-known/acme` needs to be accessible for Let's Encrypt request validation
